optimize/self-managed/optimize-deployment/migration-update/3.9-to-3.10.md

```diff
@@ -0,0 +1,23 @@
+---
+id: 3.9-to-3.10
+title: "Update notes (3.9.x to 3.10.x)"
+---
+
+:::note Heads up!
+To update Optimize to version 3.10, perform the steps in the [migration and update instructions](./instructions.md).
+:::
+
+The update to 3.10.x can be performed from any 3.9.x release.
+
+Here you will find information about:
```

Comment:
Is it necessary to include lines 12-14 for redundancy? If this expands, I could understand outlining what the document will step through but don't find it necessary yet.

Reply:
yeah it is potentially redundant, I will remove it

Reply:
I agree, why dont we do this in a separate ticket tho as this doesnt necessarily relate to the navbar styling configuration?

Reply:
Because you are adding the migration page (this is the first thing of note being added). The migration page should always have this information, even if in some cases in the past it might not have. I think we should include it here as doing it in a separate ticket seems redundant

Reply:
sorry for the delay - just implemented the changes :)

Reply:
@christinaausley let me know if there is something else we should still add from your perspective

Reply:
This looks good to me!